infrastructure/main.tf

```diff
@@ -73,6 +73,7 @@ module "sscs-cor-frontend" {
     S2S_URL                                        = "${local.s2sUrl}"
     S2S_SECRET                                     = "${data.azurerm_key_vault_secret.sscs-s2s-secret.value}"
     MYA_FEATURE_FLAG                               = "${var.mya_feature_flag}"
+    POST_BULK_SCAN                                 = "${var.post_bulk_scan}"
```

Comment:
It does not seem this feature is turned on AAT for AKS

Reply:
Overridden in flux